src/js/spreadsheet/Class/helpers/addRow.js

```diff
@@ -0,0 +1,27 @@
+// addRow.js
+import Cell from '../Cell';
+import numberToLetter from '../../../helpers/numberToLetter';
+import cellRow from '../../cellRow';
+
+export default function addRow(cols, row) {
+  // function to add a row to the spreadsheet
+
+  const newRow = [];
+
+  for (let i = 0; i < cols; i++) {
+    newRow.push(
+      new Cell({
+        row: row,
+        col: i,
+        id: `${numberToLetter(i)}${row + 1}`,
+      }),
+    );
+  }
+  const spreadSheetContainer = document.querySelector('.spreadsheet-container');
```

Comment:
Rows should be appended to `<tbody>` inside the table. Then the first cell in the appended row will be sticky as well.
```suggestion
  const spreadSheetContainer = document.querySelector('.spreadsheet-container tbody');
```

Reply:
Thank you! 💯